### DISCUSSION TITLE
微调数据库格式问题

### DISCUSSION THREAD

**Discussion by Beyond0831** [QwenLM/Qwen3#1050] [Q&A]
![image](https://github.com/user-attachments/assets/d1629ac7-e7e7-4474-bf10-d4afe152e4d0)

为什么在# 使用LLaMA-Factory微调Qwen模型的 介绍文件（finetune-zh.md) 中提到应该将数据库储存为jsonl的格式，但是实际上llamafactory 的webui里面好像只接受json格式文件呢？ 
![image](https://github.com/user-attachments/assets/87128fb0-7704-4824-a0a4-a3e10a5d212a)
尝试使用jsonl的数据库无法在webui中调用

**Comment by jklj077** [comment]
ping @yangjianxin1
